integration-cli/check_test.go

```diff
@@ -207,6 +207,7 @@ func (s *DockerSwarmSuite) AddDaemon(c *check.C, joinSwarm, manager bool) *Swarm
 		Daemon: NewDaemon(c),
 		port:   defaultSwarmPort + s.portIndex,
 	}
+	d.Daemon.disableIPTables = true
 	d.listenAddr = fmt.Sprintf("0.0.0.0:%d", d.port)
 	err := d.StartWithBusybox()
```

Comment:
You could just change this line, it already takes parameters:

` d.StartWithBusybox("--iptables=false")`